docs/source/pandas_df_conversion.rst

```diff
@@ -0,0 +1,252 @@
+
+        **Parameters:**
+
+        **long :** default False
+        
+        set to True if you want to have a long/expanded/melted dataframe with one row per peak. Faster but
+        replicated RT information. If False, returns rows in the style: rt, np.array(mz), np.array(int)
+        
+        **Returns:**
+
+        **pandas.DataFrame** 
+        
+        peak map information stored in a DataFrame
+
+**Examples:**
+
+.. code-block:: python
+
+    urlretrieve(url+'BSA1.mzML', 'BSA1.mzML')
+    exp = MSExperiment()
+    MzMLFile().load('BSA1.mzML', exp)
+
+    df = exp.get_df() # default: long = False
+    df.head(2)
+    
+.. csv-table:: exp.get_df()
+   :widths: 2 10 50 50
+   :header: ,"RT", "mzarray", "intarray"
+
+   "0",	"1501.41394", "[300.0897645621494, 300.18132740129533, 300.20...",	"[3431.0261, 1181.809, 1516.1746, 1719.8547, 11..."
+   "1", "1503.03125", "[300.06577092599525, 300.08932376441896, 300.2...",	"[914.79034, 1842.2311, 2395.1025, 851.4738, 16..." 
+
+
+.. code-block:: python
+
+    df = exp.get_df(long=True)
+    df.head(2)
+
+.. csv-table:: exp.get_df(long=True)
+   :widths: 2 20 20 20
+   :header: , "RT",	"mz", "inty"
+
+   "0", "1501.41394", "300.089752",	"3431.026123"
+   "1",	"1501.41394",	"300.181335",	"1181.808960"
+
+FeatureMap
+**********
+
+**pyopenms.FeatureMap.get_df(** *meta_values = None* **)**
+        Generates a pandas DataFrame with information contained in the FeatureMap.
+
+        **Parameters:**
+
+        **meta_values :** default None
+        
+        meta values to include (None, [custom list of meta value names] or 'all')
+        
+        **Returns:**
+
+        **pandas.DataFrame** 
+        
+        feature information stored in a DataFrame
+
+**Examples:**
+
+.. code-block:: python
+
+    urlretrieve(url+'BSA1_F1.featureXML', 'BSA1_F1.featureXML')
+    feature_map = FeatureMap()
+    FeatureXMLFile().load('BSA1_F1.featureXML', feature_map)
+
+    df = feature_map.get_df() # default: meta_values = None
+    df.head(2)
+    
+.. csv-table:: feature_map.get_df()
+   :widths: 2 20 5 20 20 20 20 20 20 20 20
+   :header: 	, "sequence",	"charge",	"RT",	"mz",	"RTstart",	"RTend",	"mzstart",	"mzend",	"quality",	"intensity"
+            "id"
+
+   "9650885788371886430",	"None",	"2",	"1942.600083",	"395.239277",	"1932.484009",	"1950.834351",	"395.239199",	"397.245758",	"0.808494",	"157572000.0"
+   "18416216708636999474",	"None",	"2",	"1749.138335",	"443.711224",	"1735.693115",	"1763.343506",	"443.71112",	"445.717531",	"0.893553",	"54069300.0"
+
+
+.. code-block:: python
+
+    df = feature_map.get_df(meta_values = 'all')
+    df.head(2)
+
+.. csv-table:: feature_map.get_df(meta_values = 'all')
+   :widths: 2 20 5 20 20 20 20 20 20 20 20 20 20 20 20 20 20
+   :header: 	, "sequence",	"charge",	"RT",	"mz",	"RTstart",	"RTend",	"mzstart",	"mzend",	"quality",	"intensity", "FWHM", "spectrum_index", "spectrum_native_id", "label", "score_correlation",	"score_fit"
```

Comment:
what is sequence here?

Reply:
it is the sequence of the best hit peptide for this feature

Reply:
ok great. 
- maybe also have the getScore() of that hit?
- maybe rename mzstart to MZstart /end

Reply:
Hm, then it would be better to update the pyOpenMS code first before releasing this tutorial?